### PR TITLE
Add one more parameter(keyboardType) to AlertIOS.promot().

### PR DESCRIPTION
**Pull request by yueshuaijie**
Add one more parameter which can specify the keyboard type of first text input(if exists) to AlertIOS.prompt().

Example usage: 
`AlertIOS.prompt('Type a phone number', null, null, 'plain-text', undefined, 'phone-pad')`

### CODE REVIEW

**Comment by facebook-github-bot-6** [comment]
By analyzing the blame information on this pull request, we identified @caabernathy and @corbt to be potential reviewers.

**Comment by facebook-github-bot-4** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @caabernathy as a potential reviewer. Could you take a look please or cc someone with more context?

**Comment by hramos** [comment]
This seems pretty useful. Did you confirm that this rendered correctly within the UIExplorer? Do you have any screenshots?

If you can resolve the conflicts, that would unblock this from being merged.

**Comment by yueshuaijie** [comment]
@hramos Yes, it works with the UIExplorer. I add an example to it.

File: Libraries/Alert/AlertIOS.js [hunk 1]
```diff
@@ -244,6 +249,7 @@ class AlertIOS {
       defaultValue,
       cancelButtonKey,
       destructiveButtonKey,
+      keyboardType
```
Comment (hramos) [inline Libraries/Alert/AlertIOS.js]: Add a comma at the end of this line. We follow this convention to ensure we don't need to update the preceding line when adding parameters.

**Comment by hramos** [comment]
One minor nit.

File: Libraries/Alert/AlertIOS.js (hunk 1 above)
Comment (yueshuaijie) [inline Libraries/Alert/AlertIOS.js]: Thanks! Patch updated.

**Comment by evantahler** [comment]
This is great! did this make it into v0.40?

**Comment by hramos** [comment]
@evantahler no, as this PR landed 4 days ago, it didn't make the cut for 0.40. It will be picked up by the 0.42 Release Candidate (February) when that is cut next month.

**Comment by evantahler** [comment]
Thank you for the information.  I'll monkey-patch until then :D

**Pull request closed without merging**